providers/task.rb

```diff
@@ -157,7 +161,7 @@ def load_current_resource
     if task_hash[:ScheduledTaskState] == 'Enabled'
       @current_resource.enabled = true
     end
-    @current_resource.cwd(task_hash[:Folder])
+    @current_resource.cwd(task_hash[:StartIn]) unless task_hash[:StartIn] == 'N/A'
```

Comment:
no. `folder` is the path in the Scheduled task "explorer" gui where the task is created. Its totally different from CWD.